### PR TITLE
threads

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

**Review by andromash — COMMENTED** [review]
Good job! But task said we need to create a counter from 0 to 100, not to 50

File: pom.xml [hunk 1]
```diff
@@ -8,5 +8,68 @@
...
+                    <version>3.8.0</version>
+                    <configuration>
+                        <source>${jdk.version}</source>
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-war-plugin</artifactId>
```
Comment (Sofasmile) [inline pom.xml]: should we use maven-war-plugin here?

File: src/main/java/thread/Counter.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+public class Counter {
+    private static final int MAX_VALUE = 100;
+    private int value;
+
+    public int getValue() {
+        return value;
+    }
+
+    public static int getMaxValue() {
```
Comment (Sofasmile) [inline src/main/java/thread/Counter.java]: better remove static

**Review by Sofasmile — COMMENTED** [review]
good job

File: pom.xml (hunk 1 above)
Comment (julia-shvabovich) [reply pom.xml]: nope

File: src/main/java/thread/Counter.java (hunk 2 above)
Comment (julia-shvabovich) [reply src/main/java/thread/Counter.java]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**